DESCRIPTION

```diff
@@ -1,29 +1,41 @@
 Package: iheiddown
 Title: A package for writing IHEID documents in RMarkdown
-Date: 2020-12-19
-Version: 0.7.4
+Date: 2021-02-18
+Version: 0.8.0
```

Comment:
You need to push a commit with the following commit message for the auto-version-bump to be at the minor level: `#minor bump ...`